app/models/unpublishing.rb

```diff
@@ -14,8 +16,22 @@ class Unpublishing < ApplicationRecord
   validates :edition, presence: true, uniqueness: true
   validates :type, presence: true, inclusion: { in: VALID_TYPES }
   validates :explanation, presence: true, if: :withdrawal?
-  validates :alternative_path, presence: true, if: :redirect?
-  validates_with UnpublishingRedirectValidator
+  validates :redirects, presence: true, if: :redirect?
+  validates_with RoutesAndRedirectsValidator, document_type: :type, if: :redirect?
```

Comment:
We can't pass the `base_path` here as we don't know anything about the instance (`type` worked as we passed the name of the method which was then called in `RoutesAndRedirectsValidator`, but we can't name the method as `:edition.base_path`). I think the `base_path` delegate is not too bad, you could argue that an unpublishing has a `base_path`, it just happens to be the same as the edition it was before it became unpublished.